### ISSUE TITLE
Tutorial/Example for OptimizationProblem

### ISSUE THREAD

**Issue by aaron-skydio** [symforce-org/symforce#111]

**Comment by jescap** [comment]
Are you planning having an tutorial/example for an MPC-like formulation?

**Comment by aaron-skydio** [comment]
Yes - the OptimizationProblem class is useful for managing large problems in general, particularly when the size is known at code generation time, including MPC but also many of the other problems that SymForce is designed to solve.  We do plan to add both examples of OptimizationProblem (which may be examples of any of those problem types) and of MPC or other control or trajectory optimization problems
